turn/src/server/mod.rs

```diff
@@ -25,22 +30,21 @@ pub struct Server {
     realm: String,
     channel_bind_timeout: Duration,
     pub(crate) nonces: Arc<Mutex<HashMap<String, Instant>>>,
-    shutdown_tx: Mutex<Option<watch::Sender<bool>>>,
+    handle: Mutex<Option<broadcast::Sender<Command>>>,
```

Comment:
This name feels awfully generic compared to the previous `shutdown` name, is there a specific reason for the change?

Reply:
Ah, I think I got confused then